app/src/main/java/org/mozilla/fenix/compose/ComposeUtil.kt

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Let's rename this file to `ComposeUtils` to be consistent with most of our `Utils` naming